### ISSUE TITLE
Setting axis and figure parameters in `ferriteviewer`

### ISSUE THREAD

**Issue by asinghvi17** [Ferrite-FEM/FerriteViz.jl#128]
It would be cool to pass through `axis` and `figure` kwargs in `ferriteviewer`, so we can configure things like aspect ratios and labels!

**Comment by koehlerson** [comment]
yeah definitely, it may also be worth to think about composeability with the new SpecAPI
